ql/src/java/org/apache/hadoop/hive/ql/io/orc/OrcSplit.java

```diff
@@ -94,7 +93,7 @@ protected OrcSplit() {
   public OrcSplit(Path path, Object fileId, long offset, long length, String[] hosts,
       OrcTail orcTail, boolean isOriginal, boolean hasBase,
       List<AcidInputFormat.DeltaMetaData> deltas, long projectedDataSize, long fileLen, Path rootDir,
-      OffsetAndBucketProperty syntheticAcidProps) {
+      OffsetAndBucketProperty syntheticAcidProps) throws IOException {
```

Comment:
please remove `throws exception` from the `parseBaseOrDeltaBucketFilename` declaration and here.